src/index.js

```diff
@@ -11,17 +11,15 @@ import Dragon from './components/Dragons/Dragon';
 import MissionList from './components/MissionList';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <Provider store={store}>
-      <BrowserRouter>
-        <Navigation />
-        <Routes>
-          <Route path="/" element={<Rockets />} />
-          <Route path="/missions" element={<MissionList />} />
-          <Route path="/Dragons" element={<Dragon />} />
-        </Routes>
-      </BrowserRouter>
-    </Provider>
-  </React.StrictMode>,
+  <Provider store={store}>
+    <BrowserRouter>
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<Rockets />} />
+        <Route path="/missions" element={<MissionList />} />
+        <Route path="/Dragons" element={<Dragon />} />
+      </Routes>
+    </BrowserRouter>
+  </Provider>,
   document.getElementById('root'),
 );
```

Comment:
Dear @MeqdamAlqudah ,

Kindly explain why you had to remove the **<React.StrictMode>** from index.js file.

Reply:
we don't need React.StrictMode here but If you want me to keep it sure😀